spec/requests/v1/submissions_spec.rb

```diff
@@ -96,12 +96,14 @@
 
   describe 'POST /submissions/:submission_id/entries' do
     it 'stores a submission entry, not associated with a file' do
-      submission = FactoryBot.create(:submission)
+      framework = FactoryBot.create(:framework, short_name: 'RM3756')
+      submission = FactoryBot.create(:submission, framework: framework)
 
       params = {
         data: {
           type: 'submission_entries',
           attributes: {
+            entry_type: 'invoice',
```

Comment:
this reminds me, we really should add code to enforce entry_type always being present on entries...